<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>624</x>
     <y>6</y>
     <width>170</width>
     <height>588</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QCheckBox" name="ckHexReceive">
      <property name="text">
       <string>16进制接收</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ckHexSend">
      <property name="text">
       <string>16进制发送</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ckAscii">
      <property name="text">
       <string>控制字符</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ckShow">
      <property name="text">
       <string>暂停显示</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ckDebug">
      <property name="text">
       <string>模拟设备</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ckAutoSend">
      <property name="text">
       <string>定时发送</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cboxInterval"/>
    </item>
    <item>
     <widget class="QLabel" name="labBindIP">
      <property name="text">
       <string>绑定地址</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cboxBindIP"/>
    </item>
    <item>
     <widget class="QLabel" name="labBindPort">
      <property name="text">
       <string>绑定端口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtBindPort"/>
    </item>
    <item>
     <widget class="QLabel" name="labServerIP">
      <property name="text">
       <string>服务器地址</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtServerIP"/>
    </item>
    <item>
     <widget class="QLabel" name="labServerPort">
      <property name="text">
       <string>服务器端口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtServerPort"/>
    </item>
    <item>
     <widget class="QPushButton" name="btnConnect">
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnClear">
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="spacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="txtMain">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>289</y>
     <width>611</width>
     <height>271</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>572</y>
     <width>612</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layTcpClient">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QComboBox" name="cboxData">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSend">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labBindPort_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>154</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>调试界面</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_x">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>速度x</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_y">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>速度y</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_z">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>速度alpha</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_xVel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_yVel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_zVel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>步态模式 </string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_GaitMode">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>对脚步态(90度)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>三脚步态(90度)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>对脚步态(180度)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>三脚步态(180度)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>外弧面缓步步态(90度)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>下爬</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>转弯</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_gaitMode">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_p1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>14</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>下压</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_p2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>14</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>35</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 6pt &quot;Adobe 宋体 Std L&quot;;
color: rgb(255, 85, 0);</string>
   </property>
   <property name="text">
    <string>LF</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>35</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 6pt &quot;Adobe 宋体 Std L&quot;;
color: rgb(255, 85, 0);</string>
   </property>
   <property name="text">
    <string>RH</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>35</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 6pt &quot;Adobe 宋体 Std L&quot;;
color: rgb(255, 85, 0);</string>
   </property>
   <property name="text">
    <string>RF</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_p4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>14</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>周期</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>35</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 6pt &quot;Adobe 宋体 Std L&quot;;
color: rgb(255, 85, 0);</string>
   </property>
   <property name="text">
    <string>LH</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spb_GaitPeriod">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_p3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>14</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_gaitPeriod">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_press">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_runOrstop">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
